gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```

Comment:
Please revert this change; we were already using Gradle `4.10.2`.

Reply:
I just move the line. it catch the eye when the last line is the longest. more easy to catch the version. 
May  I keep it here?

Reply:
fair enough :+1: